### ISSUE TITLE
Mystic Adept Power point Karma cost wrong at chargen

### ISSUE THREAD

**Issue by chummer5a** [chummer5a/chummer5a#97]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_
_Originally opened as https://github.com/chummer5a/chummer5/issues/23_

----

```
1. Create a mystic adept.
2. Grant them a power point under common.
3. Karma is decreased by 5 instead of 2.

>Mystic adepts must purchase their Power Points with Karma (2 Karma each at 
character creation for a full Power Point, with a maximum number of points 
equal to their Magic attribute rating). SR5 p69.
```

Original issue reported on code.google.com by `Yukonese...@gmail.com` on 17 Nov 2014 at 10:02

**Comment by chummer5a** [comment]
<a href="https://github.com/GoogleCodeExporter"><img src="https://avatars.githubusercontent.com/u/9614759?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [GoogleCodeExporter](https://github.com/GoogleCodeExporter)**
_Tuesday Mar 17, 2015 at 08:56 GMT_

----

```
Power Points do indeed cost 5 Karma now. See the core rulebook errata, p.1: 
"The amount of karma mystic adepts spend to buy Power Points is changed from 2 
Karma to 5 Karma per full Power Point."
```

Original comment by `butterla...@gmail.com` on 19 Nov 2014 at 3:38

**Comment by chummer5a** [comment]
Resolved in build 5.155.
